tests/terraform/sqs_delete/main.tf

```diff
@@ -0,0 +1,7 @@
+provider "aws" {
```

Comment:
please annotate new files in the custodian repo with our standard spdx license and copyright header.

Reply:
updated across the patchset :+1: